app/src/pages/playground/PlaygroundDatasetExamplesTable.tsx

```diff
@@ -215,7 +618,15 @@ export function PlaygroundDatasetExamplesTable({
     return colSizes;
     // eslint-disable-next-line react-compiler/react-compiler
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [table.getState().columnSizingInfo, table.getState().columnSizing]);
+  }, [
+    // eslint-disable-next-line react-compiler/react-compiler
```

Comment:
not sure why eslint is making me put these in all of these places

Reply:
I don't think it likes the inline calls to `table.getState()`

I think if you pull these vars out just above the useEffect it will be less angry as it can track the references.

```
const {columnSizingInfo, columnSizing} = table.getState()
```

Reply:
ahh yeah i know why it's angry, but i was just annoyed that eslint made me suppress the warning three times. I'm not sure why the other table resizing was done like this, Maybe i can pull them out like you said and it will still work. Will give it a shot